command/server.go

```diff
@@ -1076,11 +1096,37 @@ func (c *ServerCommand) Run(args []string) int {
 		default:
 			devStorageType = "inmem"
 		}
-		config, err = server.DevConfig(devStorageType)
+
+		if c.flagDevTLS {
+			if c.flagDevTLSCertDir != "" {
+				_, err := os.Stat(c.flagDevTLSCertDir)
+				if err != nil && !os.IsNotExist(err) {
```

Comment:
Should this error when the directory does not exist as well?

Reply:
Good catch, fixed.